### PR TITLE
StartOS concepts maintenance

### PR DESCRIPTION
**Pull request by kn0wmad**

### CODE REVIEW

File: site/source/learn/concepts/start9-startos.rst [hunk 1]
```diff
@@ -10,21 +10,25 @@ Start9 Server & StartOS
 
 .. _server:
 
+Server Lite
+-----------
+Server Lite is a plug 'n' play hardware device, based on the `Raspberry Pi <https://www.raspberrypi.org/>`_, assembled in-house by Start9.  The current iteration features an SD card, speaker, and passive cooling in a custom case. Lite comes pre-loaded with :ref:`StartOS<start-os>`.  For more details, check out the `Server Lite tech specs <https://store.start9.com/products/server-lite>`_.
```
Comment (Dominion5254) [inline site/source/learn/concepts/start9-startos.rst:15]:
```suggestion
Server Lite is a plug 'n' play hardware device, based on the `Raspberry Pi <https://www.raspberrypi.org/>`_, assembled in-house by Start9.  The current iteration features an SD card, speaker, and passive cooling. The Server Lite comes pre-loaded with :ref:`StartOS<start-os>`.  For more details, check out the `Server Lite tech specs <https://store.start9.com/products/server-lite>`_.
```

File: site/source/learn/concepts/start9-startos.rst [hunk 2]
```diff
@@ -10,21 +10,25 @@ Start9 Server & StartOS
...
+
 Server One
 ----------
-Server One is a plug 'n' play hardware device, based on the `Raspberry Pi <https://www.raspberrypi.org/>`_, assembled in-house by Start9.  The current iteration features an internal SSD, speaker, active cooling, and power management in a custom AIO (all-in-one) case. The One comes pre-loaded with :ref:`StartOS<start-os>`.  For more details, check out the `Server One tech specs <https://store.start9.com/products/server-one>`_.
+Server One is a plug 'n' play hardware device, based on the `Intel NUC <https://en.wikipedia.org/wiki/Next_Unit_of_Computing>`_, assembled in-house by Start9.  The current iteration features an internal SSD, speaker, active cooling, and power management in a custom AIO (all-in-one) case. One comes pre-loaded with :ref:`StartOS<start-os>`.  For more details, check out the `Server One tech specs <https://store.start9.com/products/server-one>`_.
```
Comment (Dominion5254) [inline site/source/learn/concepts/start9-startos.rst:19]:
```suggestion
Server One is a plug 'n' play hardware device, based on the `Intel NUC <https://en.wikipedia.org/wiki/Next_Unit_of_Computing>`_, assembled in-house by Start9.  The current iteration features an internal SSD, speaker, active cooling, and power management in a custom AIO (all-in-one) case. The Server One comes pre-loaded with :ref:`StartOS<start-os>`.  For more details, check out the `Server One tech specs <https://store.start9.com/products/server-one>`_.
```

File: site/source/learn/concepts/start9-startos.rst [hunk 3]
```diff
@@ -10,21 +10,25 @@ Start9 Server & StartOS
...
 
 Server Pro
 ----------
-Server Pro is a plug 'n' play hardware device, based on the `Librem Mini <https://puri.sm/products/librem-mini/>`_, built by `Purism <https://puri.sm/about/>`_.  The Pro features a custom-built circuit board with active cooling, and an open-source firmware / boot stack. The Pro comes pre-loaded with :ref:`StartOS<start-os>`.  For more details, check out the `Server Pro tech specs <https://store.start9.com/products/server-pro>`_.
+Server Pro is a plug 'n' play hardware device, based on the `Librem Mini <https://puri.sm/products/librem-mini/>`_, built by `Purism <https://puri.sm/about/>`_.  Pro features a custom-built circuit board with active cooling, and an open-source firmware / boot stack. Pro comes pre-loaded with :ref:`StartOS<start-os>`.  For more details, check out the `Server Pro tech specs <https://store.start9.com/products/server-pro>`_.
```
Comment (Dominion5254) [inline site/source/learn/concepts/start9-startos.rst:23]:
```suggestion
Server Pro is a plug 'n' play hardware device, based on the `Librem Mini <https://puri.sm/products/librem-mini/>`_, built by `Purism <https://puri.sm/about/>`_.  Pro features a custom-built circuit board with active cooling, and an open-source firmware / boot stack. The Server Pro comes pre-loaded with :ref:`StartOS<start-os>`.  For more details, check out the `Server Pro tech specs <https://store.start9.com/products/server-pro>`_.
```

**Review by Dominion5254 — CHANGES_REQUESTED** [review]
Edit for the Server Lite cases (not all are in a custom case) and some minor grammar changes.

**Comment by kn0wmad** [comment]
The flirc cases are also "custom," and my understanding is we don't use the word "the" in front of products, but to save time I'm committing everything you changed for now

**Pull request merged**